psyneulink/library/compositions/pytorchcomponents.py

```diff
@@ -108,15 +148,15 @@ def log_value(self):
             self._mechanism.output_port.parameters.value._set(detached_value, self._context)
             self._mechanism.parameters.value._set(detached_value, self._context)
 
-    def _gen_llvm_execute_derivative_func(self, ctx, builder, state, params, arg_in):
+    def _update_llvm_param_gradients(self, ctx, builder, state, params, node_delta_w, z_value, backpropagated_error, *, tags=frozenset()):
```

Comment:
Why are you adding a `tags` argument? should the import below use `tags.union({"derivative"})?
The function signature as a whole does not match the parent prototype.

Reply:
yes, the intent was to union the derivative tag. will fix

Reply:
actually, I think i can write this in a more general form (removing the long list of args) to begin with. I will take a stab at this.

Reply:
nvm - this will require more work than previously thought.

for now, I have updated the `_update_llvm_param_gradients` function:
1) standardized signature
2) remove tags pass in (wasn't ever passed in anyways)
3) removed `*` from signature (shouldn't be necessary)